src/topp/FalseDiscoveryRate.cpp

```diff
@@ -89,8 +89,21 @@ class TOPPFalseDiscoveryRate :
     setValidStrings_("PSM", ListUtils::create<String>("true,false"));
     registerStringOption_("peptide", "<FDR level>", "false", "Perform FDR calculation on peptide level and annotates it as meta value\n(Note: if set, also calculates FDR/q-value on PSM level.)", false);
     setValidStrings_("peptide", ListUtils::create<String>("true,false"));
+    registerStringOption_("PSM_peptide_base_score", "<score name or type>", "", "Set if you want to choose a different score than the last calculated main score for PSM or peptide level.", false);
```

Comment:
@jpfeufer might need some additional documentation. why PSM and peptide? etc.

Reply:
Well it obviously depends on the parameter right before it. Not sure if that needs to made any clearer.